src/common/locations.ts

```diff
@@ -186,3 +196,48 @@ export function getColleges(fips: string): CollegeData[] {
 export function getStateCode(stateName: string) {
   return REVERSED_STATES[stateName];
 }
+
+const ALL_LOCATIONS = getLocationNames();
+const [COUNTIES, STATES] = partition(ALL_LOCATIONS, isCounty);
+
+export function getStateByUrlName(stateUrlName: string): Location | undefined {
+  return STATES.find(
+    state =>
+      toLower(state.state_url_name) === toLower(stateUrlName) ||
+      toLower(state.state_code) === toLower(stateUrlName),
+  );
+}
+
+export function getCountyByUrlName(
+  countyUrlName: string,
```

Comment:
I think we need to always use the stateUrlName and countyUrlName together, since county URL names aren't unique.  E.g. these both end up showing the same stats right now:

https://covid-projections-git-pablo-update-location-page-url.covidactnow.vercel.app/us/or/county/washington_county?s=1083238
https://covid-projections-git-pablo-update-location-page-url.covidactnow.vercel.app/us/mn/county/washington_county?s=1083238

Reply:
good catch! It's working correctly now, I added the `stateCode` to the function